internal/scan-manager/webapp/pom.xml

```diff
@@ -97,6 +97,36 @@
             <artifactId>httpmime</artifactId>
             <version>${org.apache.httpcomponents.httpclient.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Shall we please add `maven-java-doc` plugin to this project with this PR. Add the commented configuration at https://github.com/wso2/security-tools/blob/master/internal/pom.xml#L264

We can incrementally add it to other projects.